index.js

```diff
@@ -0,0 +1,52 @@
+const title = document.querySelector('.title_input');
+const author = document.querySelector('.author_input');
+const empty = document.querySelector('.empty_bookstore');
+const form = document.querySelector('.book_form');
+const tableBody = document.querySelector('.table_body');
+const storedBooks = JSON.parse(localStorage.getItem('bookstored'));
```

Comment:
- [ ] See the error I get when I try to add a book for the first time. You probably need to clear your local storage to replicate this error.

  ![image](https://user-images.githubusercontent.com/81468675/211546269-0846dcaf-aa1a-4c69-911c-8884f6478b38.png)

  You can fix this by initializing `storedBooks` to an empty array in the event it is `null`:

  ```
  const storedBooks = JSON.parse(localStorage.getItem('bookstored')) || [];
  ```

Reply:
Thank you @julie-ify. This went unnoticed but we have seen the error and addressed it.